src/Tribe/Rewrite.php

```diff
@@ -193,18 +193,35 @@ public function remove_percent_placeholders( array $rules ) {
 		return $rules;
 	}
 
+	/**
+	 * Filter for the `rewrite_rules_array` hook.
+	 *
+	 * @since TBD
+	 *
+	 * @param array|mixed $rules The rules to be filtered.
+	 *
+	 * @return array|mixed Rules after filtering.
+	 */
+	public function filter_rewrite_rules_array( $rules ) {
+		if ( ! is_array( $rules ) ) {
+			return $rules;
```

Comment:
Is there a good reason to not simply cast it to an array?

Reply:
It's supposed to be one: https://github.com/WordPress/wordpress-develop/blob/6.1/src/wp-includes/class-wp-rewrite.php#L190

Reply:
Even if we cast it, it will be malformed. I don't see a scenario where casting it will solve anything other than pushing the problem farther downstream?

i.e. 
```
$rules = 'bob';
return (array)$rules; // ['bob'] is not a legit structure (right?)
```

Reply:
Fair enough.

Reply:
Haha, ja man!